pkg/load/load.go

```diff
@@ -363,6 +364,10 @@ func Registry(root string, flat bool) (registry.ReferenceByName, registry.ChainB
 				observers[observer.Observer.Name] = observer.Observer.Observer
 			} else if strings.HasSuffix(path, CommandsSuffix) {
 				// ignore
+			} else if filepath.Base(path) == config.ConfigVersionFileName {
+				if version, err := ioutil.ReadFile(path); err != nil {
+					logrus.WithField("version", string(version)).Info("Resolved configuration version")
```

Comment:
I think you're missing one of the branches?  This should be the `else` branch.

Reply:
:trollface: